### PR TITLE
Set docker.wso2.com registry domain

### LINKED ISSUE #94: Set domain docker.wso2.com
**Description:**
Change Docker registry from "dockerhub.wso2.com" to "docker.wso2.com".

### PR DESCRIPTION
**Pull request by ThilinaManamgoda**
fixes #94

### CODE REVIEW

File: docker-compose/APIM-ISasKM-with-Analytics/README.md [hunk 1]
```diff
@@ -13,9 +13,11 @@
   *  [WSO2 API Manager Analytics Dockerfile](../../dockerfiles/apim-analytics/README.md)
   *  [WSO2 IS as KM](../../dockerfiles/is-as-km/README.md)
  
-  > In the `docker-compose.yml`, remove the `dockerhub.wso2.com/` prefix from the `image` name
+  > In order to run the Docker Compose configuration, you will need a Subscription from WSO2 since the Docker images hosted at docker.wso2.com contains the latest updates and fixes to WSO2 API Manager 2.1.0, API Manager Analytics 2.1.0 and WSO2 Identity Server as KM 5.3.0. You can sign up for a Free Trial Subscription here[https://wso2.com/free-trial-subscription]
```
Comment (msmshariq) [inline docker-compose/APIM-ISasKM-with-Analytics/README.md]: The "here" should be a hyperlink and url should point to .com.

File: docker-compose/APIM-ISasKM-with-Analytics/README.md [hunk 2]
```diff
@@ -13,9 +13,11 @@
   *  [WSO2 API Manager Analytics Dockerfile](../../dockerfiles/apim-analytics/README.md)
   *  [WSO2 IS as KM](../../dockerfiles/is-as-km/README.md)
  
-  > In the `docker-compose.yml`, remove the `dockerhub.wso2.com/` prefix from the `image` name
+  > In order to run the Docker Compose configuration, you will need a Subscription from WSO2 since the Docker images hosted at docker.wso2.com contains the latest updates and fixes to WSO2 API Manager 2.1.0, API Manager Analytics 2.1.0 and WSO2 Identity Server as KM 5.3.0. You can sign up for a Free Trial Subscription [here](https://wso2.com/free-trial-subscription)
```
Comment (chirangaalwis) [inline docker-compose/APIM-ISasKM-with-Analytics/README.md]: @ThilinaManamgoda shall we make the word 'Subscription' start with simple 's' in order to be more consistent?

File: docker-compose/APIM-with-Analytics/README.md [hunk 3]
```diff
@@ -11,9 +11,12 @@
 1. Build the WSO2 API manager 2.1.0 and API Manager Analytics 2.1.0 Docker images:
      *  [WSO2 API Manager Dockerfile](../../dockerfiles/apim/README.md)
      *  [WSO2 API Manager Analytics Dockerfile](../../dockerfiles/apim-analytics/README.md)
-    > In the `docker-compose.yml`, remove the `dockerhub.wso2.com/` prefix from the `image` name
+     
+    > In order to run the Docker Compose configuration, you will need an active Subscription from WSO2 since the Docker images hosted at docker.wso2.com contains the latest updates and fixes to WSO2 API manager 2.1.0 and API Manager Analytics 2.1.0. You can sign up for a Free Trial Subscription [here](https://wso2.com/free-trial-subscription)
```
Comment (chirangaalwis) [inline docker-compose/APIM-with-Analytics/README.md]: @ThilinaManamgoda shall we make the word 'Subscription' start with simple 's' in order to be more consistent?

File: docker-compose/APIM-with-Analytics/README.md [hunk 4]
```diff
@@ -4,36 +4,35 @@
...
     ```
 
-4. Execute the following Docker Compose command to start the deployment:
+3. Execute the following Docker Compose command to start the deployment:
     ```
     docker-compose up
     ```
 
-5. Add the following entrie to the /etc/hosts.
+4. Add the following entrie to the /etc/hosts.
```
Comment (chirangaalwis) [inline docker-compose/APIM-with-Analytics/README.md]: This has to be made `entries`.

**Pull request merged**